Lib/test/test_ast.py

```diff
@@ -2564,6 +2565,33 @@ def test_subinterpreter(self):
         self.assertEqual(res, 0)
 
 
+class ASTMainTests(unittest.TestCase):
+    """Tests `ast.main()` function."""
```

Comment:
AFAIK `unittest` prints docstrings out.

```suggestion
    # Tests `ast.main()` function.
```